app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt

```diff
@@ -23,7 +26,51 @@ data class Wallpaper(
 /**
  * A type hierarchy representing the different theme collections [Wallpaper]s belong to.
  */
-sealed class WallpaperThemeCollection {
-    object None : WallpaperThemeCollection()
-    object Firefox : WallpaperThemeCollection()
+enum class WallpaperThemeCollection(val origin: WallpaperOrigin) {
+    NONE(WallpaperOrigin.LOCAL),
+    FIREFOX(WallpaperOrigin.LOCAL),
+    FOCUS(WallpaperOrigin.REMOTE),
+}
+
+/**
+ * Get the parent directory name for a remote wallpaper asset.
+ */
+fun WallpaperThemeCollection.directoryName(): String = when (this) {
+    WallpaperThemeCollection.NONE,
+    WallpaperThemeCollection.FIREFOX -> ""
+    WallpaperThemeCollection.FOCUS -> "focus"
+}
```

Comment:
nit: This could be a property on the `WallpaperThemeCollection` :) 
We could do that in follow up.

Reply:
A better name might actually be what we want here. This function returns the remote directory name, so I don't think it's necessarily an applicable property for all theme collections